charts/victoria-metrics-auth/values.yaml

```diff
@@ -184,8 +184,8 @@ serviceMonitor:
 #    tlsConfig:
 #      insecureSkipVerify: true
 
-# -- Use existing configmap if specified otherwise .config values will be used. Ref: https://victoriametrics.github.io/vmauth.html
-configMap: ""
+# -- Use existing secret if specified otherwise .config values will be used. Ref: https://victoriametrics.github.io/vmauth.html
```

Comment:
Could you mention, that `Secret` must contain key `auth.yml` with valid vmauth configuration?